conda_lock/src_parser/poetry.py

```diff
@@ -0,0 +1,98 @@
+import collections
+import collections.abc
+import copy
+import pathlib
+
+from typing import List, Optional
+
+import requests
+import toml
+import yaml
+
+from conda_lock.src_parser import LockSpecification
+
+
+# TODO: make this configurable
+PYPI_TO_CONDA_NAME_LOOKUP = "https://raw.githubusercontent.com/marcelotrevisani/grayskull/master/grayskull/pypi/config.yaml"
```

Comment:
We need a stable place for this in conda-forge. Grayskull should read from that. A simple repo with that file would be useful in many projects.

Reply:
yeah we're planning on having the bot autogenerate this